omod-1.8/pom.xml

```diff
@@ -76,6 +76,14 @@
 					<header>${project.parent.basedir}/license-header.txt</header>
 				</configuration>
 			</plugin>
-		</plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
```

Comment:
Why this change?

Reply:
This was automatically generated on changing the pom.xml file, I though this auto-generated code was necessary. Should I Remove this?

Reply:
Yes, remove it.